script/test/fixtures/multiple-compose-files/output-openshift.json

```diff
@@ -255,19 +275,35 @@
                 ],
                 "env": [
                   {
+                    "name": "MYSQL_ROOT_PASSWORD",
+                    "value": "kubernetes"
```

Comment:
Is this config correct? Because same env is created twice? Will this run on a k8s cluster?

Seems like something libcompose is not handling?